src/contracts/libraries/GPv2TradeExecution.sol

```diff
@@ -22,6 +23,10 @@ library GPv2TradeExecution {
     address internal constant BUY_ETH_ADDRESS =
         0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
 
+    /// @dev Marker address used to indicate that the receiver of the trade
+    /// proceeds should the owner of the order.
+    address internal constant RECEIVER_SAME_AS_OWNER = address(0);
```

Comment:
Cute to use address for this indicator instead of bool, I suppose having the same type makes some logical gates easier to work with.

Reply:
Its mostly done because of the EIP-712 signing, and "burning" can be implemented by sending to `address(1)` (a precompile).